client/abstract-client.js

```diff
@@ -205,11 +217,9 @@ class AbstractClient {
             throw Error("Unable to get results, need handler id");
         }
         let searchResponse = {};
-        let retries = 0;
-        let timeout = options.timeout ? options.timeout : this.defaultTimeoutInSeconds;
-        let numberOfResults = options.numberOfResults
-            ? options.numberOfResults
-            : this.defaultNumberOfResults;
+        let timeoutInSeconds = options.timeoutInSeconds ? options.timeoutInSeconds : this.defaultTimeoutInSeconds;
```

Comment:
this.defaultTimeoutInSeconds should be this.timeoutInSeconds since it will be defined as defaul in constructor if not specified

Reply:
same for line 221

Reply:
Correct, but we should have ability to customize it per function or particular call

Reply:
Sorry, my error